### PR TITLE
Issue #4313: Split and Organize Checkstyle inputs by Test for checks in annotation package: AnnotationUseStyle

### PR DESCRIPTION
**Pull request by Kietzmann**
#4313

### CODE REVIEW

**Comment by Kietzmann** [comment]
Fixed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleCustomAnnotations.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
+
+@MyAnnotation11 @MyAnnotation12 @MyAnnotation13
+public class InputAnnotationUseStyleCustomAnnotations {
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleCustomAnnotations.java]: I don't see this input being used anywhere, so why are we copying it over?

**Comment by Kietzmann** [comment]
@rnveach 
Fixed

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationsUseStyleParams.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.checks.annotation;
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
```
Comment (rnveach) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationsUseStyleParams.java]: This class should start with `InputAnnotationUseStyle...`. It has an `s` after `Annotation`.

Comment (Kietzmann) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationsUseStyleParams.java]: Fixed

**Comment by Kietzmann** [comment]
@rnveach 
FIxed

**Pull request merged**